cookbook/cookbook/src/shared-components/list-ingredients-component/list-ingredients-component.js

```diff
@@ -1,19 +1,50 @@
 import React, {Component} from 'react';
+import './list-ingredients-component.css';
 
 export class ListIngredientsComponent extends Component {
+  onCheckboxClick = id => e => {
+    const {recept, triggerIngridientState} = this.props;
+    console.log('recept', recept);
+    console.log('id ingredient', id)
+    triggerIngridientState(recept.id, id);
+  };
+
   render() {
     const{
-      ingredients
+      recept,
+      viewMode
     } = this.props;
-    //console.log('ListIngredientsComponent props', this.props);
-    return(
-      <ol>
-        {
-          ingredients && ingredients.map(ingredient => {
-            return (<li>{ingredient.name}</li>)
-          })
-        }
-      </ol>
-    )
-  };
+
+    if(viewMode === true) {
+      return (
+        <ol>
+          {
+            recept.ingredients && recept.ingredients.map((ingredient, index) => {
+              return (<li key={ingredient.name + index}>{ingredient.name} ( {ingredient.quantity} )</li>)
+            })
+          }
+        </ol>
+      )
+    } else {
+      return(
+        <ol>
+          {
+            recept.ingredients && recept.ingredients.map((ingredient, index) => {
```

Comment:
it worth to move it to method something like `renderReducer`

You should omit to have a logic inside JSX